tests/components/harmony/test_commands.py

```diff
@@ -0,0 +1,263 @@
+"""Test sending commands to the Harmony Hub remote."""
```

Comment:
Our standard is to divide the tests in modules that correspond to the platforms the integration supports plus config flow and init module tests. Ie the switch platform should have tests in a module named `test_switch.py` etc.